pkg/client/clientset_generated/clientset/typed/servicecatalog/v1alpha1/serviceinstance_expansion.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1alpha1"
+)
+
+// The ReferencesExtension interface allows setting the References
```

Comment:
One thing that I realized this weekend is that we now need to remove or rewrite the `purge-generated` target in the Makefile, because it will remove this code.

Reply:
Created https://github.com/kubernetes-incubator/service-catalog/issues/1309 for this